File: TapeEquilibrium/php/TestTapeEquilibrium.php
<?php
require 'TapeEquilibrium.php';

class Test extends PHPUnit_Framework_TestCase {

	protected $tape_equilibrium;

	protected function setup() {
		$this->tape_equilibrium = new \Codility\TapeEquilibrium;
	}

	public function testSolution() {
		$this->assertEquals(1, $this->tape_equilibrium->solution([3,1,2,4,3]));
	}

}

File: TapeEquilibrium/ruby/TestTapeEquilibrium.rb
require_relative 'TapeEquilibrium'
require 'minitest/autorun'

class Test < MiniTest::Test

	def setup
		@tape_equilibrium = TapeEquilibrium.new
	end

	def test_solution
		assert_equal 1, @tape_equilibrium.solution([3,1,2,4,3])
	end

end
